File: daily.py
# use cron to run this command daily:
# cd /home/USER/atop-server && /usr/local/bin/pipenv run python3 /home/USER/atop-server/daily.py 2>&1 | logger -t

from pymongo import MongoClient
client = MongoClient()
db = client.atop

from pymongo.collection import ReturnDocument

from bson.objectid import ObjectId


result = db.users.update_many({}, 
    {'$set':
        {
            'remaining_likes': 100
        }
    }
)
print('[result] users daily charging: matched_count: {}  modified_count: {}'.format(
        result.matched_count, result.modified_count
    )
)

File: daily.js
const MongoClient = require('mongodb').MongoClient
const MONGODB_URL = 'mongodb://127.0.0.1:27017';
const ObjectID = require('mongodb').ObjectID

var db
MongoClient.connect(MONGODB_URL, {
    useNewUrlParser: true
    }, (err, client) => {
    if (err) return console.error('[DAILY] ERROR mongodb: '+err)
    db = client.db('atop')
    db.collection('users').updateMany({}, {$set: {remaining_likes: 100}}, (err, r) => {
        if (err)
            console.error('[DAILY] ERROR mongodb: '+err)
        console.log('[DAILY] LOG mongodb: '+r)
        client.close()
        return;
    })
})